packages/spanner/src/database.js

```diff
@@ -577,6 +593,25 @@ Database.prototype.run = function(query, options, callback) {
  *   .on('end', function() {});
  *
  * //-
+ * // If you need to enforce a specific param type, a types map can be provided.
+ * // This is typically useful if your param value can be null.
```

Comment:
I know it seems odd, but in my system tests for querying types, if you don't specify a type for certain values the tests will fail because it assumes the wrong type. I think for the most part our users won't need to do this though